GlyssenEngineTests/Quote/QuoteParserTests.cs

```diff
@@ -5370,6 +5370,21 @@ public void Parse_MultipleInterruptionsInMiddleOfMultiBlockQuote_MultiBlockQuote
 			Assert.AreEqual(CharacterVerse.kScriptureCharacter, output[5].CharacterId);
 		}
 		#endregion
+		
+		[Test]
+		public void Parse_GetCharactersWhenReferenceTestIsMissing()
```

Comment:
The name should be in the form <<method under test>>_<<condition>>_<<result>>. So for example:
GetCharacters_ReferenceTextIsNull_ReturnsOnly GetsOnlyCharacterForVerse()
GetCharacters_ReferenceTextIsNullAndVerseHasMultiplesCharacters_GetsAllCharactersForVerse()
Since the test for 19:10:6 and 40:7:8 (character and character2) are really the same condition, you could use [TestCase(19, 10, 6)] and [TestCase(40, 7, 8)] instead of [Test] to show that clearly.

Reply:
Changed test as requested.  There are now 2 tests.